src/conmon.c

```diff
@@ -289,6 +289,7 @@ int main(int argc, char *argv[])
 	if (signal(SIGCHLD, on_sigchld) == SIG_ERR)
 		pexit("Failed to set handler for SIGCHLD");
 
+	atexit(reap_children);
```

Comment:
From looking at the documentation, it looks like multiple `atexit` functions are executed in reverse order, so this will be executed after the exit delay, thus reintroducing the Podman issue - maybe reorder this to after exit command?

Reply:
nice good catch. unfortunately, we need to run it earlier, so there'll just be multiple calls :woman_shrugging: